pkg/core/transaction/transaction.go

```diff
@@ -197,6 +197,16 @@ func (t Transaction) GroupOutputByAssetID() map[util.Uint256][]*Output {
 	return m
 }
 
+// GetSignable returns a part of the transaction which must be signed.
+func (t *Transaction) GetSignable() []byte {
```

Comment:
`GetSignedPart`? We also have `Hashable` defined here. Just a suggestion though.